### PR TITLE
[NEW] REST API endpoint `/v1/rooms.delete`

### PR DESCRIPTION
**Pull request by felipe-rod123**

### CODE REVIEW

File: packages/rest-typings/src/v1/teams/index.ts [hunk 1]
```diff
@@ -182,4 +182,8 @@ export type TeamsEndpoints = {
 			room: IRoom;
 		};
 	};
+
+	'/v1/teams.eraseRoom': {
```
Comment (ggazzo) [inline packages/rest-typings/src/v1/teams/index.ts:186]: move to room

Comment (ggazzo) [reply packages/rest-typings/src/v1/teams/index.ts:186]: /v1/room.erase

**Review by ggazzo — CHANGES_REQUESTED** [review]

**Review by ggazzo — CHANGES_REQUESTED** [review]
pleas check comments bellow

**Review by ggazzo — APPROVED** [review]

**Comment by ggazzo** [comment]
> any special reason to not follow the same pattern as `POST users.delete` ?

since we have no ~good~pattern defined I decided to follow the world convention (RestFul) and apply the http methods usage properly, maybe we could keep both (one because our ancient patterns and the right one) what do you think ?

**Comment by sampaiodiego** [comment]
my suggestion is to use the same pattern we use on other endpoints, so for this one it would be `POST rooms.delete { roomId: 'ID-TO-DELETE' }` .. I know it is not the best pattern, but it is the one we use, so it is somewhat predictable if you're already familiar with Rocket.Chat API.

**Review by sampaiodiego — DISMISSED** [review]

**Review by sampaiodiego — APPROVED** [review]

**Pull request merged**